app/models/Recommendation.php

```diff
@@ -86,8 +86,19 @@ public static function getUserRecs($id)
                     'optional_question as rec_optional_question',
                     ];
         $recommendations = self::where('application_id', $id)->select($fields)->get();
+
         if ($recommendations) {
-            return $recommendations->toArray();
+            $recommendations_array = [];
+
+            foreach ($recommendations as $recommendation) {
+                $recommendation_token = $recommendation->generateRecToken($recommendation);
```

Comment:
agh we should probably re-name `generateRecToken` at some point, since it's really a "find or create" situation, but it sounds like it would always make a new one

Reply:
not necessary at all to change in this PR, just thinking out loud!

Reply:
ah that is such a great call! Do you think I should make a card for this @katiecrane ?